packages/config/config/devices/0x027a/zen23_4.0.json

```diff
@@ -0,0 +1,238 @@
+// Zooz ZEN23
+// Z-Wave Plus On/Off Toggle Switch v4
+{
+	"revision": 1,
+	"manufacturer": "Zooz",
+	"manufacturerId": "0x027a",
+	"label": "ZEN23",
+	"description": "Z-Wave Plus On/Off Toggle Switch v4",
+	"devices": [
+		{
+			"productType": "0xb111",
+			"productId": "0x251c"
+		}
+	],
+	"firmwareVersion": {
+		"min": "4.0",
+		"max": "4.255"
+	},
+	"supportsZWavePlus": true,
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Group 2",
+			"description": "BASIC Command Set",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Toggle Control",
+			"description": "Determines whether up is on and down is off; or up is off, down is on",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "UP=On, DOWN=Off",
+					"value": 0
+				},
+				{
+					"label": "UP=Off, DOWN=On",
+					"value": 1
+				},
+				{
+					"label": "ANY=Toggle State",
+					"value": 2
+				}
+			]
+		},
+		"3": {
+			"label": "Auto Turn-Off Timer",
+			"description": "Use this parameter to enable or disable the auto turn-off timer",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Auto Turn-Off Timer Disabled",
+					"value": 0
+				},
+				{
+					"label": "Auto Turn-Off Timer Enabled",
+					"value": 1
+				}
+			]
+		},
+		"4": {
+			"label": "Auto Turn-Off Timer Time",
+			"description": "Time after which the switch automatically turns off",
+			"unit": "minutes",
+			"valueSize": 4,
+			"minValue": 1,
+			"maxValue": 65535,
+			"defaultValue": 60,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"5": {
+			"label": "Auto Turn-On Timer",
+			"description": "Use this parameter to enable or disable the auto turn-on timer",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Auto Turn-On Timer Disabled",
+					"value": 0
+				},
+				{
+					"label": "Auto Turn-On Timer Enabled",
+					"value": 1
+				}
+			]
+		},
+		"6": {
+			"label": "Auto Turn-On Timer Time",
+			"description": "Set the time after which the switch automatically turns on",
+			"unit": "minutes",
+			"valueSize": 4,
+			"minValue": 1,
+			"maxValue": 65535,
```

Comment:
I guess this is true but it seems weird that manufacturers use 4 bytes to store a 2-byte value

Reply:
Yeah, it's apparently 4-bytes from their own docs: https://www.support.getzooz.com/kb/article/317-zen23-on-off-toggle-switch-ver-4-01-advanced-settings/

> Parameter 6: Use this parameter to set the time after which you want the switch to automatically turn on once it has been turned off. The number entered as value corresponds to the number of minutes.
> Values: 1 – 65535 (minutes). Default: 60.
> Size: 4 byte dec